### DISCUSSION TITLE
v3.1.1

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#61] [公告]
修正 CI 管道构建，3.0~3.1.1 更新日志在 CI 调试完毕后汇总

<hr /><em>This discussion was created from the release <a href='https://github.com/Ghost-chu/PeerBanHelper/releases/tag/v3.1.1'>v3.1.1</a>.</em>

**Comment by zhongfly** [comment]
所以啥问题？
如果说是更新日志，https://github.com/ncipollo/release-action 文档里只有启用generateReleaseNotes才会修改，然而默认值就是false

**Reply by Ghost-chu** [reply]
尽管它被关了，但是 CI 跑完，Release Notes 就强制被改成了最后一次 commit 的内容。

以及 Docker 的 Native Image 打包好像还有点问题。

**Reply by zhongfly** [reply]
那也许是omitBodyDuringUpdate的设置。算了，我不折腾了

**Reply by zhongfly** [reply]
要删除的话，把最后一个release的job删掉就行了
